applications/GeoMechanicsApplication/custom_constitutive/coulomb_yield_surface.cpp

```diff
@@ -25,17 +25,15 @@ CoulombYieldSurface::CoulombYieldSurface(double FrictionAngleInRad, double Cohes
 {
 }
 
-double CoulombYieldSurface::YieldFunctionValue(const Vector& rPrincipalStress) const
+double CoulombYieldSurface::YieldFunctionValue(const Vector& rSigmaTau) const
 {
-    return 0.5 * (rPrincipalStress(0) - rPrincipalStress(2)) +
-           0.5 * (rPrincipalStress(0) + rPrincipalStress(2)) * std::sin(mFrictionAngle) -
-           mCohesion * std::cos(mFrictionAngle);
+    return rSigmaTau[1] + rSigmaTau[0] * std::sin(mFrictionAngle) - mCohesion * std::cos(mFrictionAngle);
 }
 
-Vector CoulombYieldSurface::DerivativeOfFlowFunction(const Vector& rPrincipalStress) const
+Vector CoulombYieldSurface::DerivativeOfFlowFunction(const Vector& rSigmaTau) const
```

Comment:
As far as I understand this is only for geo_mechanics, if yes, can we change it signature to ()? rSigmaTau is not used in this function.

Reply:
This is an implementation of the yield surface interface, where in general you'd like to receive the current stress state. Since that stress state is irrelevant for this particular implementation, I'd suggest to "unname" the function parameter:
```suggestion
Vector CoulombYieldSurface::DerivativeOfFlowFunction(const Vector&) const
```
Naturally, this should also be done in the corresponding header file.

Reply:
Done.